ArcGISRuntimeSDKQt_CppSamples/Analysis/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.cpp

```diff
@@ -0,0 +1,368 @@
+// [WriteFile Name=ConfigureSubnetworkTrace, Category=Analysis]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "ConfigureSubnetworkTrace.h"
+
+#include "CodedValueDomain.h"
+#include "TaskWatcher.h"
+#include "UtilityAssetGroup.h"
+#include "UtilityAssetType.h"
+#include "UtilityCategory.h"
+#include "UtilityCategoryComparison.h"
+#include "UtilityDomainNetwork.h"
+#include "UtilityElement.h"
+#include "UtilityElementTraceResult.h"
+#include "UtilityNetwork.h"
+#include "UtilityNetworkDefinition.h"
+#include "UtilityNetworkSource.h"
+#include "UtilityNetworkTypes.h"
+#include "UtilityTerminal.h"
+#include "UtilityTerminalConfiguration.h"
+#include "UtilityTier.h"
+#include "UtilityTraceAndCondition.h"
+#include "UtilityTraceConfiguration.h"
+#include "UtilityTraceOrCondition.h"
+#include "UtilityTraceParameters.h"
+#include "UtilityTraceResultListModel.h"
+#include "UtilityTraversability.h"
+
+#include <QQmlEngine>
+
+using namespace Esri::ArcGISRuntime;
+
+ConfigureSubnetworkTrace::ConfigureSubnetworkTrace(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_busy(true),
+  m_dialogVisible(false),
+  m_textFieldVisible(true),
+  m_assetGroupName(QString("Circuit Breaker")),
+  m_assetTypeName(QString("Three Phase")),
+  m_deviceTableName(QString("Electric Distribution Device")),
+  m_domainNetworkName(QString("ElectricDistribution")),
```

Comment:
Initializing QString with `(QString("someText"))` is redundant.

```suggestion
  m_domainNetworkName("ElectricDistribution"),
```

Same suggestion for all of these.